package.json

```diff
@@ -143,41 +143,85 @@
       "type": "object",
       "title": "Ansible",
       "properties": {
+        "ansibleServer.trace.server": {
+          "type": "string",
+          "enum": [
+            "off",
+            "messages",
+            "verbose"
+          ],
+          "default": "off",
+          "description": "Traces the communication between VSCode and the ansible language server.",
+          "order": 0
+        },
         "ansible.ansible.path": {
           "scope": "machine-overridable",
           "type": "string",
           "default": "ansible",
-          "description": "Path to the ansible executable. All subcommands are expected to have adjacent locations."
+          "description": "Path to the ansible executable. All subcommands are expected to have adjacent locations.",
+          "order": 1
         },
         "ansible.ansible.useFullyQualifiedCollectionNames": {
           "scope": "resource",
           "type": "boolean",
           "default": true,
-          "description": "Always use fully qualified collection names (FQCN) when inserting a module name. Disabling it will only use FQCNs when necessary."
+          "description": "Always use fully qualified collection names (FQCN) when inserting a module name. Disabling it will only use FQCNs when necessary.",
+          "order": 2
         },
-        "ansible.ansibleLint.arguments": {
-          "scope": "resource",
+        "ansible.python.interpreterPath": {
+          "scope": "machine-overridable",
+          "type": "string",
+          "default": "python3",
+          "description": "Path to the Python interpreter executable. Particularly important if you are using a Python virtual environment. Leave blank to use Python from PATH.",
+          "order": 3
+        },
+        "ansible.python.activationScript": {
+          "scope": "machine-overridable",
           "type": "string",
           "default": "",
-          "description": "Command line arguments to be passed to ansible-lint."
+          "description": "Path to the virtual environment activation script. Use only if you have a custom activation script. It will be sourced using bash before executing Ansible commands. When set, the Interpreter Path setting is ignored.",
+          "order": 4
         },
-        "ansible.ansibleLint.enabled": {
+        "ansible.ansibleNavigator.path": {
+          "default": "ansible-navigator",
+          "description": "%configuration.navigate.executablePath%",
+          "scope": "machine-overridable",
+          "type": "string",
+          "order": 5
+        },
+        "ansible.validation.enabled": {
+          "type": "boolean",
+          "default": true,
+          "description": "Toggles the validation provider.",
```

Comment:
Please add more description here. For eg:
`Toggles the validation provider. If the enabled it will run all the validation providers that are individually enabled, for example if  ansible.validation.lint.enabled is `true` it will run `ansible-lint`. It none of the provider is enabled it runs `ansible-playbook --syntax-check` to provide basic validations.

Reply:
That proposed description looks to long to me. I might clutter the configuration screen.

But I agree that original one, should be updated, let's find something in between.

Reply:
Here is the PR addressing this: https://github.com/ansible/vscode-ansible/pull/649